node/src/blockchain/blockchain_bridge.rs

```diff
@@ -323,21 +323,19 @@ impl BlockchainBridge {
                 _ => so_far,
             });
         let (vector_of_results, error_opt) = short_circuit_result;
-        if !vector_of_results.is_empty() {
-            let pairs = vector_of_results
-                .into_iter()
-                .zip(msg.pending_payable.iter().cloned())
-                .collect_vec();
-            self.payment_confirmation
-                .report_transaction_receipts_sub_opt
-                .as_ref()
-                .expect("Accountant is unbound")
-                .try_send(ReportTransactionReceipts {
-                    fingerprints_with_receipts: pairs,
-                    response_skeleton_opt: msg.response_skeleton_opt,
-                })
-                .expect("Accountant is dead");
-        }
+        let pairs = vector_of_results
```

Comment:
Ask Bert if we ned to modify any code while handling this message inside the `scanners.rs`.